config/v1/types_network.go

```diff
@@ -51,6 +51,11 @@ type NetworkSpec struct {
 	// not allowed to be set.
 	// +optional
 	ExternalIP *ExternalIPConfig `json:"externalIP,omitempty"`
+
+	// Service Node port range for services to be exposed on that are of type
+	// Spec.Type = NodePort. If empty, the kube-apiserver is configured
+	// with the default range (30000-32767).
```

Comment:
If you're feeling fancy, you could write a regex that validates this format.

Reply:
done.